source/shared/core_stmt.cpp

```diff
@@ -2501,6 +2528,12 @@ void resize_output_buffer_if_necessary( _Inout_ sqlsrv_stmt* stmt, _Inout_ zval*
 
     // account for the NULL terminator returned by ODBC and needed by Zend to avoid a "String not null terminated" debug warning
     SQLULEN field_size = column_size;
+    // with AE on, when column_size is retrieved from SQLDescribeParam, column_size does not include the decimal place
+    // include the decimal for output params by adding elem_size
+    if ( stmt->conn->ce_option.enabled && decimal_digits != 0 )
+    {
```

Comment:
is decimal_digits normally >= 0? so should it be > 0 here instead? Or even better, make this a boolean flag instead to add extra elem_size or not